_data/goforbg/goforbg.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Bharadwaj Giridhar
+institution: PSNA College of Engineering. 
+profile_pic: goforbg.jpg 
+quote: "It's never a bug, it's a feature" 
```

Comment:
Please follow the given [instruction](https://github.com/education/graduation#3-add-your-profile-information) by removing the quotes to maintain the same format